### DISCUSSION TITLE
Request access to the GitHub Student Developer Pack without a student card

### DISCUSSION THREAD

**Discussion by Sali-Mami** [community/community#70744] [GitHub Education]
### Select Topic Area

Question

### Body

I am currently a student, and I would like to access the GitHub Student Developer Pack to enhance my programming skills and contribute to open-source projects.
I would greatly appreciate any guidance or advice on how to proceed in this case.

Thank you in advance for your assistance and guidance.

**Comment by Nahid4306053** [comment]
Certainly! Accessing the GitHub Student Developer Pack is a great way to enhance your programming skills and contribute to open-source projects. Here's a step-by-step guide on how to proceed:

1. Verify Your Student Status:
Before you can access the GitHub Student Developer Pack, you need to verify your student status. GitHub partners with SheerID, a third-party verification service, to confirm your eligibility. You can do this by going to the [GitHub Student Developer Pack page](https://education.github.com/pack) and clicking on the "Get your Pack" button. Follow the prompts to complete the verification process.

2. Prepare Necessary Documentation:
You'll likely need to provide some form of documentation to prove your student status. This can include a school-issued email address, school ID card, or any official document that confirms you are currently enrolled in an educational institution.

3. Create a GitHub Account:
If you don't already have a GitHub account, you'll need to create one. Visit [GitHub's homepage](https://github.com/) and sign up for a new account using your preferred email address.

4. Apply for the GitHub Student Developer Pack:
Once you've verified your student status and created a GitHub account, you can apply for the GitHub Student Developer Pack. Follow the steps on the GitHub Student Developer Pack page to fill out the application form. Be sure to provide accurate information and the necessary documentation.

5. Wait for Approval:
After you submit your application, GitHub will review it. This process might take a few days. Make sure to check your email regularly, as GitHub might contact you for additional information or to notify you of your approval.

6. Explore the Pack:
Once your application is approved, you'll gain access to a variety of tools and resources offered in the GitHub Student Developer Pack. These can include free access to developer tools, cloud hosting credits, domain registration credits, and more. Explore the offerings and choose the ones that align with your interests and goals.

7. Contribute to Open-Source Projects:
With the resources from the GitHub Student Developer Pack, you can enhance your programming skills and contribute to open-source projects. GitHub hosts a vast number of open-source projects covering a wide range of technologies. You can find projects that match your interests and start contributing to them. Remember to read the project's guidelines and documentation before making contributions.

8. Learn and Network:
In addition to accessing free tools, GitHub offers various educational resources. Take advantage of tutorials, guides, and courses available on GitHub and other platforms to continue learning and improving your skills. Additionally, engage with the GitHub community, participate in discussions, and network with other developers.

Remember that the GitHub Student Developer Pack is a valuable resource, but your learning and contributions are what truly enhance your skills and make a difference in the open-source community. Best of luck with your programming journey! 🚀

**Reply by Dkver** [reply]
sir i am uploaded my id before login and verification with student email address . how i am remove my id from github to again login .please help

**Reply by Nahid4306053** [reply]
If you've uploaded your ID for verification on GitHub and want to remove it, you'll need to contact GitHub support for assistance. They are the only ones who can help you with account verification and the removal of sensitive information. Here's how you can reach GitHub support:

Go to GitHub's Help page: Visit the GitHub Help page at https://support.github.com.

Choose a relevant topic: Click on a relevant topic that matches your issue or question. If you can't find an exact match, select a related topic.

Search for your issue: Use the search bar to find articles related to your problem. Often, GitHub provides detailed guides on common issues.

If you can't find a solution: If you can't find the information you need, scroll down to the bottom of the page and look for the "Still can't find what you're looking for?" section. There, you'll usually find an option to contact support.

Contact GitHub Support: Click on the "Contact us" or "Submit a request" button. This will allow you to send a message to GitHub's support team.

When contacting GitHub support, provide them with as much detail as possible about your issue, including the fact that you want to remove your ID from the verification process. They will guide you through the necessary steps and assist you in resolving the problem. Remember to be patient, as it might take some time for them to respond to your request.

**Comment by Sheikhlipu123** [comment]
If you don't have Student Card, use academic transcript, or other document showing proof of your academic status.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Sali-Mami , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by devashish2024** [comment]
For me too. I am enrolled in governemnt school, India's government and even private schools do not provide a email or website of school, but a gmail ofcourse, my school not even gave a ID card. I have no such proof also.

**Comment by darshansharma** [comment]
@ashishagarwal2023 They do private library cards or Identity cards. To confirm your status as a student, you are required to provide photographs showing both the front and rear views of the relevant card.

**Reply by devashish2024** [reply]
I'm in a government school, meaning no fees. However, the India government tries to improve the school and the school staff eats most of the money - in my school there hasn't been a ID card I joined 2 years ago and none yet. They provide tie and belt tho

edit: they provide meals at lunch thought

**Reply by darshansharma** [reply]
what is the name of the school? Do you pay any fees to school? Do you give exams in school or do you go to school daily? 
You can submit an application signed by your headmaster or any teacher who is teaching there that I am studying in the [ name of school ] since [ your joining date ] and my course duration will last till [ date of completion of course ] and I want to use this application for purpose of taking [ scholarship/reimbursment/give it to government ] (mention any one) and tell them to stamp it. 

It will act as a proof of your identity.

**Reply by devashish2024** [reply]
Ok, thought nice idea.
